localsettings.example.py

```diff
@@ -1,13 +1,13 @@
 from dev_settings import *
 
-####### Database config #######
+"""####### Database config #######"""
```

Comment:
Nit: I'd prefer to either ignore the warnings for this, or to configure the linter with a rule that ignores them. I'd also imagine that something like:
`# ###### Database config ######`
skips the warning and isn't so jarring